core/domain/takeout_service_test.py

```diff
@@ -379,6 +386,8 @@ def test_export_data_trivial(self):
             'collection_progress_data': collection_progress_data,
             'story_progress_data': story_progress_data,
             'general_feedback_thread_data': general_feedback_thread_data,
+            'general_feedback_thread_user_data':
```

Comment:
Do

```
    'general_feedback_thread_user_data': (
        general_feedback_thread_user_data)
```

The naming between this field and the previous one is a bit confusing though. Is there any way to disambiguate it more clearly?

Reply:
Because of the way we're generating the names (manipulating the string of the model name) there isn't really a great way to disambiguate more clearly without adding an explicit exception for this particular model. I feel like it is better to leave it as is, as this is the name generated by our flexible system of manipulating model.__name__